mp/src/game/shared/momentum/mom_system_gamemode.cpp

```diff
@@ -122,9 +123,13 @@ void CGameMode_KZ::SetGameModeVars()
 {
     CGameModeBase::SetGameModeVars();
 
-    // KZ-specific
+    // KZ-specific (Based on SimpleKZ variables)
+    sv_accelerate.SetValue(6.5f);
     sv_airaccelerate.SetValue(100);
-    sv_maxspeed.SetValue(250);
+    sv_friction.SetValue(5.2f);
+    sv_maxspeed.SetValue(320);
+    sv_maxvelocity.SetValue(3500);
+    sv_wateraccelerate.SetValue(10);
```

Comment:
Need default wateraccelerate set inside base gamemode

Reply:
thanks for pointing it out, thought i had removed it since default wateracc is 10